go.mod

```diff
@@ -0,0 +1,26 @@
+module github.com/IBM/satellite-object-storage-plugin
+
+go 1.15
+
+replace (
+	k8s.io/api => k8s.io/api v0.0.0-20190516230258-a675ac48af67
+	k8s.io/apimachinery => k8s.io/apimachinery v0.0.0-20190313205120-d7deff9243b1
+	k8s.io/apiserver => k8s.io/apiserver v0.0.0-20190313205120-8b27c41bdbb1
+)
+
+require (
+	github.com/IBM/ibm-cos-sdk-go v1.6.0
+	github.com/IBM/ibmcloud-volume-interface v1.0.0-beta4
+	github.com/container-storage-interface/spec v1.2.0
+	github.com/ctrox/csi-s3 v1.1.1
+	github.com/golang/glog v0.0.0-20160126235308-23def4e6c14b
+	github.com/kubernetes-csi/drivers v1.0.2
+	github.com/mitchellh/go-ps v0.0.0-20170309133038-4fdf99ab2936
+	github.com/prometheus/client_golang v1.9.0
+	github.ibm.com/alchemy-containers/ibm-csi-common v1.0.0-beta08
```

Comment:
it complains for this one when I execute `go mod download`
it returns:
```
go: github.ibm.com/alchemy-containers/ibm-csi-common@v1.0.0-beta08: reading github.ibm.com/alchemy-containers/ibm-csi-common/go.mod at revision v1.0.0-beta08: unknown revision v1.0.0-beta08
```

Reply:
Could you try after setting below ENV vars
```
export GO111MODULE=on
export GONOPROXY=github.ibm.com
export GOSUMDB=off
```

Reply:
Same effect :(
```
[yiannis@oc3656886087 satellite-object-storage-plugin]$ go mod download
go: github.ibm.com/alchemy-containers/ibm-csi-common@v1.0.0-beta08: reading github.ibm.com/alchemy-containers/ibm-csi-common/go.mod at revision v1.0.0-beta08: unknown revision v1.0.0-beta08
[yiannis@oc3656886087 satellite-object-storage-plugin]$ export GOSUMDB=off
[yiannis@oc3656886087 satellite-object-storage-plugin]$ go mod download
go: github.ibm.com/alchemy-containers/ibm-csi-common@v1.0.0-beta08: reading github.ibm.com/alchemy-containers/ibm-csi-common/go.mod at revision v1.0.0-beta08: unknown revision v1.0.0-beta08
[yiannis@oc3656886087 satellite-object-storage-plugin]$ 
```

Reply:
@YiannisGkoufas @ambiknai 
We should not use any pkgs from IBM internal private registry like
`github.ibm.com/alchemy-containers/ibm-csi-common`